pylint/lint/utils.py

```diff
@@ -63,7 +63,7 @@ def prepare_crash_report(
     return issue_template_path
 
 
-def get_fatal_error_message(filepath: str, issue_template_path: Path) -> str:
```

Comment:
Most of the time the type for path should be ``Union[Path, str]``, I don't know what is the difference with ``PosixPath`` which seems to be the default used by pyannotate.

Reply:
PyAnnotate bases this on the test suite so it seems only `str` get passed to this function in all of the test suite. I could add the `Union` but perhaps it is better to only allow `str`'s? The test suite is passing and `mypy` will now throw errors when passing anything else than `str`?